Added documentation for #9493 (#155)
mautic/mautic-documentation: pages/09.contacts/01.manage-contacts/02.contact-monitoring/docs.en.md

![](media/google-analytics-tag-assistent.png)

> ##### How to test Google Tag Manager tracking code and campaign action

Reviewer: Let's capitalise Mautic features: https://developers.google.com/style/capitalization?hl=en

```suggestion
##### How to test Google Tag Manager tracking code and Campaign action
```

Author: Done

> - Install [Tag assistent](https://chrome.google.com/webstore/detail/tag-assistant-by-google/kejbdjndbnbjgmefkgdddjlbokphdefk) and enable recording on your website 

Reviewer: ```suggestion
- Install [Tag assistant](https://chrome.google.com/webstore/detail/tag-assistant-by-google/kejbdjndbnbjgmefkgdddjlbokphdefk) and enable recording on your website 
```

Author: Done

> - Create campaign with Visits a page decision and  Send tracking event action

Reviewer: ```suggestion
- Create a Campaign with `Visits a Page` decision and `Send tracking event` action
```

Author: Done

> - Test it and see Tag assistent debug window with one Pageview request and 1 events

Reviewer: ```suggestion
- Test it and see Tag assistant debug window with one Pageview request and 1 event.
```

Author: Done
